src/Common/src/Interop/Unix/libssl/Interop.libssl.cs

```diff
@@ -109,6 +109,12 @@ internal static partial class libssl
         [DllImport(Interop.Libraries.LibSsl)]
         internal static extern void SSL_CTX_set_quiet_shutdown(SafeSslContextHandle ctx, int mode);
 
+        [DllImport(Interop.Libraries.LibSsl)]
+        internal static extern SafeSslContextHandle SSL_get_SSL_CTX(SafeSslHandle ssl);
```

Comment:
These calls should be shimmed in System.Security.Cryptography.Native.  Managed code should call into Cryptography.Native, and our native code calls into libssl.
We are actively working on adding shims now, so adding more here just adds more work for us to shim. Might as well do it fully in the beginning.